### ISSUE TITLE
allow passing an UserManager instance

### ISSUE THREAD

**Issue by pamapa** [authts/react-oidc-context#490]

**Comment by SaulBurgos** [comment]
Hi, is there a link with an example about how to use this new feature?

**Comment by pamapa** [comment]
```js
const oidcConfig = {
  userManager: new UserManager(...), /* or variable pointing to instance of UserManager */
};

ReactDOM.render(
  <AuthProvider {...oidcConfig}>
    <App />
  </AuthProvider>,
  document.getElementById("app")
);
```

**Comment by pamapa** [comment]
fixed in 2.2.0
